packages/components/modules/profiles/Members/MemberItem/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react'
+
+import { AvatarWithPlaceholder } from '@baseapp-frontend/design-system'
+
+import { Box, Button, Typography } from '@mui/material'
+import { useFragment } from 'react-relay'
+
+import { ProfileItemFragment$key } from '../../../../__generated__/ProfileItemFragment.graphql'
+import { ProfileItemFragment } from '../../graphql/queries/ProfileItem'
+import { MemberStatuses } from '../constants'
+import { capitalizeFirstLetter } from '../utils'
+import { MemberItemContainer, MemberPersonalInformation } from './styled'
+import { MemberItemProps } from './types'
+
+const MemberItem: FC<MemberItemProps> = ({
+  member,
+  memberRole,
+  status,
+  avatarProps = {},
+  avatarWidth = 40,
+  avatarHeight = 40,
+}) => {
+  const memberProfile = useFragment<ProfileItemFragment$key>(ProfileItemFragment, member)
```

Comment:
I think ideally this fragment should be related to members (aka ProfileUserRole on the backend) instead of profile fragment, we would use a MembersItemFragment. So the list query would be something like this:
```
export const UserMembersListFragment = graphql`
  fragment UserMembersListFragment on Profile
  @refetchable(queryName: "userMembersListPaginationRefetchable")
  @argumentDefinitions(
    count: { type: "Int", defaultValue: 10 }
    cursor: { type: "String" }
    orderByStatus: { type: "String", defaultValue: "custom" }
  ) {
    members(first: $count, after: $cursor, orderByStatus: $orderByStatus)
      @connection(key: "UserMembersFragment_members", filters: ["orderByStatus"]) {
      totalCount
      edges {
        node {
          ...MemberItemFragment
        }
      }
      pageInfo {
        endCursor
        hasNextPage
      }
    }
  }
`
```

them the members fragment something like:
```
export const MemberItemFragment = graphql`
fragment MemberItemFragment on Member // need to create this interface
@argumentDefinitions(avatarSize: { type: "Int", defaultValue: 100 }) {
  id
  user {
    profile {
      ...ProfileItemFragment
    }
  }
  role
  status
}
`
```
@nossila what do you think? I think we mentioned that on our last sidebar

Reply:
then if we do that, we avoid needing to pass over the `member?.role` and `member?.status` since we would define those fields on the MemberItem fragment